plugins/continuous-toolbox/src/ContinuousToolbox.js

```diff
@@ -28,6 +28,14 @@ export class ContinuousToolbox extends Blockly.Toolbox {
     const flyout = this.getFlyout();
     flyout.show(this.getInitialFlyoutContents_());
     flyout.recordScrollPositions();
+
+    this.workspace_.addChangeListener((e) => {
+      console.log('triggered');
```

Comment:
please remove console.log statement

Reply:
Sorry about that. In the future I will be more careful about checking the code before submitting the PR.